doc/bootstrap.rst

```diff
@@ -42,8 +42,8 @@ This command does a few things:
 * Creates a monitor and manager daemon for the new cluster on the
   local host.  A minimal configuration file needed to communicate with
   the new cluster is written to ``ceph.conf`` in the local directory.
-* A copy of the ``client.admin`` administrative (privileged!) secret
-  key is written to ``ceph.client.admin.keyring`` in the local directory.
+* Writes a copy of the ``client.admin`` administrative (privileged!) secret
+  key to the file ``ceph.client.admin.keyring`` in the local directory.
```

Comment:
drop "the file" for even more consistency with the surrounding bullet points?

Reply:
Agreed. I'll revise this and make a force push.